tests/functional_pooling.py

```diff
@@ -27,6 +27,18 @@ def shaparange(s):
 
 
 class TestPooling(unittest.TestCase):
+    def test_average_pooling2d(self):
+        """Test batch_to_space2d and space_to_batch2d."""
```

Comment:
Change docstring into Test average pooling 2d

Reply:
Actually I removed the docstring, it's redundant with the test name and provided no useful information.